include/abb_librws/v2_0/rws.h

```diff
@@ -396,6 +396,11 @@ namespace abb :: rws :: v2_0
      */
     static const std::string LOGOUT;
 
+    /**
+     * \brief Progress.
+     */
+    static const std::string PROGRESS;
+
```

Comment:
Please avoid defining new string constants in a header, there is no point in it.

Reply:
I think so. You can also remove the existing ones from the header, and put them in source files where they are used, to minimize changes.